### ISSUE TITLE
2.09 breaks watch connectivity with Legizmo

### ISSUE THREAD

**Issue by AMlG0** [opa334/Dopamine#475]
It appears that the changes in 2.09 blocks the ability for Legizmo to facilitate communication between the watch and phone. The Bluetooth/VPN path is unable to inject properly and it essentially bricks the watch. 

This is occurring on iPhone 14 pro on iOS 16.0, paired with an AWU2 on Legizmo 5.1.1. 

Now the watch is stuck in set up mode even after restarting it, but it is still showing up on the phone and in iCloud. There is no way to fully remove it because of the communication block, and the watch is still activation locked. It cannot communicate with the services needed to remove that lock.

I understand that users were experiencing issues with VPN profiles, but this is a pretty important thing to have functioning. Can you please look into this?

Thank you in advance for your time.

Edit: spelling

**Comment by opa334** [comment]
Yes, this is due to blocking injection into terminusd.

Unfortunately arm64 is a mess and I don't know how to properly solve it, blocking injection doesn't even seem to have stopped the crash loops of terminusd so I guess it can just be undone :/.

**Comment by TheMasterOfMike** [comment]
> Yes, this is due to blocking injection into terminusd.
> 
> Unfortunately arm64 is a mess and I don't know how to properly solve it, blocking injection doesn't even seem to have stopped the crash loops of terminusd so I guess it can just be undone :/.

(Forgive me for jumping into this issue) AMIG0 is on an iPhone 14 Pro, which is arm64e (not arm64)

Looking at the relevant change for terminusd it seems as though it was only blocked on arm64, so I'm confused how that would affect arm64e.

**Comment by AMlG0** [comment]
> Yes, this is due to blocking injection into terminusd.
> 
> Unfortunately arm64 is a mess and I don't know how to properly solve it, blocking injection doesn't even seem to have stopped the crash loops of terminusd so I guess it can just be undone :/.

Sorry I just saw your reply. I’ve been working with Lunotech on testing a couple fixes, and it appears using the native pairing method does solve this issue.

 The tweak method was failing communication paths into like nanoprefsyncdfirst, appconduitd, tccd…etc. I was able to capture a couple of really useful sysdiagnose logs which helped uncover some of this (if not most) and a fix was pushed with update 5.1.2. 
 
It’s still kind of odd though that it affected an arm64e device though.

I’ll close this out for now though as using native method/updated version of Legizmo fixes it. If others are running into issues, we can try to help them out on the Legizmo discord. 

Thanks for your time!
